yazi-core/src/manager/commands/paste.rs

```diff
@@ -21,6 +22,12 @@ impl Manager {
 		if self.yanked.cut {
 			tasks.file_cut(&src, dest, opt.force);
 
+			src.iter().for_each(|source_file| {
```

Comment:
I think it should be moved to the scheduler, https://github.com/sxyazi/yazi/blob/357f87aecbe52f1f36efff69af67558f7cf730a2/yazi-scheduler/src/file/file.rs#L28, as moving is not immediately completed (for example, when it involves moving across mount points). Some files may fail to move due to permissions or because the mount point is full, and I think such cases should not be reported as events.

Reply:
I like this idea. I tried implementing it but ran into some issues and could use some help.

# Issue 1
In the scheduler I don't seem to have access to the current tab though. The tab is present in the rename event, and I think it would need some refactoring to get access to the current tab.

# Issue 2
Publishing events from the scheduler doesn't cause the events to be visible for me. I pushed the commit 410bf4a8852ccedafb5f9f4b05d95b717d8629dd as a work-in-progress commit to show what I tried.

I could not figure this out. Any ideas on what could be done?